keps/sig-network/784-kube-proxy-component-config/README.md

```diff
@@ -51,68 +65,72 @@ This document is intended to propose a process and desired goals by which kube-p
 
 ## Proposal
 
-The idea is to conduct the process of graduation to beta in small steps in the span of at least one Kubernetes release cycle. This will be done by creating one or more alpha versions of the config with the last alpha version being copied as v1beta1 after the community is happy with it.
-Each of the sub-sections below can result in a separate alpha version release, although it will be better for users to have no more than a couple of alpha versions past v1alpha1.
-After each alpha version release, the community will gather around for new ideas on how to proceed in the graduation process. If there are viable proposals, this document is updated with an appropriate section(s) below and the new changes are introduced in the form of new alpha version(s).
-The proposed process is similar to the already successfully used one for kubeadm.
-
-### Re-encapsulate mode specific options
+The idea is to conduct the process of graduation to beta in small steps in the span of at least one Kubernetes release cycle.
+This will be done by creating one or more alpha versions of the config with the last alpha version being copied as v1beta1 after
+the community is happy with it. Each of the subsections below can result in a separate alpha version release, although it will
+be better for users to have no more than a couple of alpha versions past v1alpha1. After each alpha version release, the community
+will gather around for new ideas on how to proceed in the graduation process. If there are viable proposals, this document is
+updated with an appropriate section(s) below and the new changes are introduced in the form of new alpha version(s). The proposed
+process is similar to the already successfully used one for kubeadm.
 
 The current state of the config has proven that:
 - Some options are deemed as mode specific, but are in fact shared between all modes.
 - Some options are placed directly into KubeProxyConfiguration, but are in fact mode specific ones.
 - There are options that are shared between some (but not all) modes. Specific features of the underlying implementation are common and this happens only within the boundaries of the platform (iptables and ipvs modes for example).
-- Although legacy Linux and Windows user mode proxies are separate code bases, they have a common set of options.
 
 With that in mind, the following measures are proposed:
-- Mode specific structs are consolidated to not use fields from other mode specific structs.
-- Introduce a single combined legacy user mode proxy struct for both Linux and Windows backends.
+- Create platform subsection for platform specific fields.
+- Move backend-agnostic and platform-agnostic fields from backend section to root section.
+- Move backend-agnostic and platform-specific fields from backend section to relevant platform section.
+- Drop legacy/unused options. 
+
+### User Stories (Optional)
+
+#### Story 1
+
+#### Story 2
 
-#### Example
+### Notes/Constraints/Caveats (Optional)
```

Comment:
likewise, if you don't actually have any Notes/Contraints/Caveats, you can omit this